keras_nlp/tokenizers/sentence_piece_tokenizer.py

```diff
@@ -148,18 +172,18 @@ def __init__(
             model=proto_bytes,
             out_type=self.compute_dtype,
         )
-
         # Keras cannot serialize a bytestring, so we base64 encode the model
         # byte array as a string for saving.
-        self.proto = base64.b64encode(proto_bytes).decode("ascii")
-        self.sequence_length = sequence_length
+        self.proto = proto
```

Comment:
Should this be `self.proto = proto_bytes` actually?

Looking at the file loading case... `proto` is a file path, `proto_bytes` is the file contents, `self.proto` would get assigned to the file path on this line, in save assets we would then just write a file with filepath inside it!

Reply:
Ah yes, you're right. Fixed to `proto_bytes`